master/definitions.xml

```diff
@@ -581,7 +568,7 @@
   <attributecategory
     name='presentation'
     href='styling.html#TermPresentationAttribute'
-    presentationattributes='alignment-baseline, baseline-shift, clip, clip-path, clip-rule, color, color-interpolation, color-interpolation-filters, color-rendering, cursor, direction, display, dominant-baseline, fill, fill-opacity, fill-rule, filter, flood-color, flood-opacity, font-family, font-size, font-size-adjust, font-stretch, font-style, font-variant, font-weight, glyph-orientation-horizontal, glyph-orientation-vertical, image-rendering, letter-spacing, lighting-color, marker-end, marker-mid, marker-start, mask, opacity, overflow, paint-order, pointer-events, shape-rendering, solid-color, solid-opacity, stop-color, stop-opacity, stroke, stroke-dasharray, stroke-dashoffset, stroke-linecap, stroke-linejoin, stroke-miterlimit, stroke-opacity, stroke-width, text-anchor, text-decoration, text-rendering, transform, unicode-bidi, vector-effect, visibility, word-spacing, writing-mode'/>
```

Comment:
I guess I should have left this in? Is the cursor attribute the same as the CSS cursor property?

Reply:
Yes, we still need to define somewhere that you can specify the cursor style property as a presentation attribute.

Reply:
added it back